pages/windows/shutdown.md

```diff
@@ -0,0 +1,35 @@
+# shutdown
+
+> A tool for shutting down, restarting or logging off a machine.
+
+- Shutdown the current machine:
+
+`shutdown /s`
+
+- Restart the current machine:
+
+`shutdown /r`
+
+- Hibernate the current machine:
+
+`shutdown /h`
+
+- Log off the current machine:
+
+`shutdown /l`
+
+- Specify a timeout in seconds to wait before shutting down:
+
+`shutdown /s /t {{seconds}}`
+
+- Specify a comment for the shutdown reason:
+
+`shutdown /s /c "{{reason}}"`
+
+- Abort a shutdown sequence:
```

Comment:
I am guessing this aborts an active shutdown when the timeout is yet to expire ? In that case, I think we can clarify it like "Abort a shutdown sequence whose timeout is yet to expire"